### ISSUE TITLE
IPv6 Support for Marzban-Node

### ISSUE THREAD

**Issue by Ashaxer** [Gozargah/Marzban#1347]
In main.py file inside Marzban-Node docker container, "0.0.0.0" is used as host in uvicorn.run()
Please make it configurable with .env (just like main Marzban) or change it to "::" to make it listen to IPv6 network Interface too.

**Comment by Ashaxer** [comment]
I can't PR, I'm newbie in these things. I cannot even fix it on my own server...
It will be appreciated if you do it.
Thank you!

**Comment by Ashaxer** [comment]
Yes, I couldn't manage to activate IPv6 for it, have you?
Tried adding setting ```SERVICE_HOST: "::"``` inside docker-compose.yml but no luck..

**Comment by Ashaxer** [comment]
Thanks
Tried that but there is another issue, when I try to set the node address with IPv6 Address I get these errors in Marzban:
[Errno -9] Address family for hostname not supported
![image](https://github.com/user-attachments/assets/2a9a6c73-19a8-4649-99d6-65b521b4b6e4)

**Comment by Ashaxer** [comment]
> Did you tested adding your IPv6 IP Address with brackets? `[2a03:xxx:5001]`

Yes indeed, it does not even recognize it as a valid address and throughs same error (Error 2) as if your input is "Hello"

**Comment by ImMohammad20000** [comment]
As i told before marzban not supporting ipv6 now we will support it in the feuture

**Comment by Ashaxer** [comment]
Hello again.
I managed to test your method by using socat command (since Marzban does not support ipv6 nodes comminucations) but looks like there is more things to do...
basically I made docker listen to ipv6 interface using these lines in docker-compose.yml
```
    ports:
      - "[::]:62050:62050"
      - "[::]:62051:62051"
      - "0.0.0.0:7004:7004"
      - "0.0.0.0:7005:7005"
      - "0.0.0.0:7006:7006"
      - "0.0.0.0:7007:7007"
```
using socat, I forwarded a local port like this inside the main server:
127.0.0.1:42050 ==> [NODE IPV6]:62050: ```socat TCP4-LISTEN:42050,bind=127.0.0.1,reuseaddr,fork TCP6:[IPv6]:62050```
127.0.0.1:42051 ==> [NODE IPV6]:62051: ```socat TCP4-LISTEN:42051,bind=127.0.0.1,reuseaddr,fork TCP6:[IPv6]:62051```

then I successfully connected to the node using 127.0.0.1:42050/42051 address.

but here is the catch
the node cannot connect to internal hosts!
I'm running a rathole tunnel on a local port but the docker cannot connect to localhost (removed network_mode: host)
tried:
```
    extra_hosts:
      - "localhost:host-gateway"
      - "127.0.0.1:host-gateway"
```
no luck... my node is a tunnel server and I forgot to test the docker connection to internet but certainly interanet didn't work 🤷🏻‍♂️
